developerDocs/advanced-use-cases.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Advanced Use Cases
+category: 64cbb5277b5f3c0065d96616
+slug: opensea-sdk-advanced-use
+parentDocSlug: opensea-sdk
+order: 2
+hidden: false
+---
+
+- [Scheduling Future Listings](#scheduling-future-listings)
+- [Purchasing Items for Other Users](#purchasing-items-for-other-users)
+- [Using ERC-20 Tokens Instead of Ether](#using-erc-20-tokens-instead-of-ether)
+- [Private Auctions](#private-auctions)
+- [Listening to Events](#listening-to-events)
+
+## Advanced
+
+Interested in purchasing for users server-side or with a bot, scheduling future orders, or making bids in different ERC-20 tokens? OpenSea.js can help with that.
+
+### Scheduling Future Listings
+
+You can create sell orders that aren't fulfillable until a future date. Just pass in a `listingTime` (a UTC timestamp in seconds) to your SDK instance:
+
+```typescript
+const auction = await openseaSDK.createSellOrder({
+  tokenAddress,
+  tokenId,
+  accountAddress,
+  startAmount: 1,
+  listingTime: Math.round(Date.now() / 1000 + 60 * 60 * 24), // One day from now
+});
+```
+
+### Purchasing Items for Other Users
+
+You can buy and transfer an item to someone else in one step! Just pass the `recipientAddress` parameter:
+
+```typescript
+const order = await openseaSDK.api.getOrder({ side: "ask", ... })
+await openseaSDK.fulfillOrder({
+  order,
+  accountAddress, // The address of your wallet, which will sign the transaction
+  recipientAddress // The address of the recipient, i.e. the wallet you're purchasing on behalf of
+})
+```
+
+If the order is a sell order (`order.side === "ask"`), the taker is the _buyer_ and this will prompt the buyer to pay for the item(s) but send them to the `recipientAddress`. If the order is a buy order ( `"bid"`), the taker is the _seller_ but the bid amount be sent to the `recipientAddress`.
+
+This will automatically approve the assets for trading and confirm the transaction for sending them.
+
+### Using ERC-20 Tokens Instead of Ether
+
+Here's an example of listing the Genesis CryptoKitty for $100! No more needing to worry about the exchange rate:
+
+```typescript
+// Token address for the DAI stablecoin, which is pegged to $1 USD
+const paymentTokenAddress = "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359";
+
+// The units for `startAmount` and `endAmount` are now in DAI, so $100 USD
+const auction = await openseaSDK.createSellOrder({
```

Comment:
`listing` not `auction`, same comment as https://github.com/ProjectOpenSea/opensea-js/pull/1137/files/b4661cc7f6fe4ccf4b36701df03b2bf91e30b2f0#r1287261645